src/test/scala/ironoxide/FullIntegrationTest.scala

```diff
@@ -62,16 +63,37 @@ class FullIntegrationTest extends AsyncWordSpec with Matchers with EitherValues
     "Create valid group" in {
       val sdk = IronSdkSync[IO](deviceContext)
       val name = GroupName("a name")
-      // this is all under the same user now, so this may not be a real test if it already exists, though it won't fail
-      val id = GroupId(java.util.UUID.randomUUID.toString)
-      val groupCreateResult = sdk.groupCreate(GroupCreateOpts(id, name)).attempt.unsafeRunSync.value
+      val groupCreateResult = sdk.groupCreate(GroupCreateOpts(validGroupId, name)).attempt.unsafeRunSync.value
 
       groupCreateResult.id.id.length shouldBe 36
       groupCreateResult.name.get.name shouldBe name.name
       groupCreateResult.isAdmin shouldBe true
       groupCreateResult.isMember shouldBe true
       groupCreateResult.created should not be null
       groupCreateResult.lastUpdated shouldBe groupCreateResult.created
+      groupCreateResult.needsRotation shouldBe Some(false)
+    }
+  }
+
+  "Group Get" should {
+    "Return data for valid group admin" in {
+      val sdk = IronSdkSync[IO](deviceContext)
+      val groupGetResult = sdk.groupGetMetadata(validGroupId).attempt.unsafeRunSync.value
+
+      groupGetResult.id.id.length shouldBe 36
+      groupGetResult.name.get.name shouldBe "a name"
+      groupGetResult.isAdmin shouldBe true
+      groupGetResult.isMember shouldBe true
+      groupGetResult.adminList shouldBe Some(List(primaryTestUserId))
```

Comment:
I think you can use `.value` on admin list to not assert about `Some(...)`

Reply:
Done